slides/08-Platform-Hub/1070-WhatDoYouThink.md

```diff
@@ -0,0 +1,5 @@
+## We love feedback
+
+Please give us feedback, this is a realtively new application so there's going to be teething issues. However, we are driven to sorting as many of them as we can and improving how projects interact with ACP platform. The following is the best place to make new issues https://github.com/UKHomeOffice/platform-hub/issues
```

Comment:
Not sure if the open source software repository is the right place for feedback (I'm assuming most of the feedback will around user workflows and set up issues). @jon-shanks and I discussed using the #platform-hub channel on Slack, at least initially. That way people can participate in the conversations and then we can triage and make relevant tasks / issues when needed.

Reply:
Can we not create a new template in the hub and it be submitted currently as a BAU support request?

Reply:
@jon-shanks – certainly can! But it'll cost ya 10p.

Reply:
The issue with "bugs / problems with the application" is a) users shouldn't be burdened with figuring out whether something is a software issue or a UX issue or an XXXX issue – so we need a single channel/place in which all feedback can be submitted and then we should triage and figure out what changes that result in.

Reply:
So I'm pushing up a change which will probably close this chat but we can discuss this evening.